src/LoggerBase.php

```diff
@@ -40,7 +20,7 @@ protected function interpolate($message, array $context = []): string
      *     corresponding placeholders.
      * @return string The resulting string.
      */
-    private function interpolateString(string $message, array $context = []): string
+    protected function interpolate(string $message, array $context = []): string
     {
         // Build a replacement array with braces around the context keys.
         $replace = [];
```

Comment:
Out of curiosity, would it be better to use the following if statement in the foreach?
```
if ($value instanceof \Stringable) {
```

Is there a reason why we don't allow objects even if they have a `__toString` method?

Reply:
I suppose that's equivalent. I'm not the best PHP programmer to be answering that question.

Good suggestion about allowing Stringable messages.